spec/unit/views/components/index_table_for_spec.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 require "rails_helper"
+require "active_admin/batch_actions/resource_extension"
+require "active_admin/batch_actions/views/selection_cells"
```

Comment:
These requires were needed to be able to run this spec individually. Running it individually was already prevented by missing constant errors before any of the other changes I made. I'll remove them if you don't care about this.

Reply:
Thank you. Yes, please remove them. Preferred to match other specs and to preload for parallel test suite run.